src/editor/components/scenegraph/SceneGraph.js

```diff
@@ -377,25 +122,10 @@ export default class SceneGraph extends React.Component {
       hide: this.state.leftBarHide
     });
 
-    const clearFilter = this.state.filter ? (
-      <a onClick={this.clearFilter} className="button fa fa-times" />
-    ) : null;
-
     return (
       <div id="scenegraph" className="scenegraph">
         <div className="scenegraph-toolbar">
           <ToolbarWrapper />
-          <div className="search">
```

Comment:
this wasn't being used at the moment, so i decided to remove all the search functionality. can add it back if we decide we need it